install_config/install/deploy_router.adoc

```diff
@@ -1113,6 +1113,73 @@ If you used a non-default name for the router, you must change *_dc/router_*
 accordingly.
 
 
+[[deploy-router-protecting-against-ddos-attacks]]
+=== Protecting Against DDoS Attacks
+
+Add *timeout http-request* to the default HAProxy router image to
+protect the deployment against distributed denial-of-service (DDoS) attacks (for
+example, slowloris):
+
+====
+----
+# and the haproxy stats socket is available at /var/run/haproxy.stats
+global
+  stats socket ./haproxy.stats level admin
+
+defaults
+  option http-server-close
+  mode http
+  timeout http-request 5s
+  timeout connect 5s <1>
+  timeout server 10s
+  timeout client 30s
+----
+<1> *timeout http-request* is set up to 5 seconds. HAProxy gives a client five
+seconds to send its whole HTTP request. Otherwise, HAProxy shuts the connection
+with an error.
+====
+
+Also, when the environment variable `*ROUTER_SLOWLORIS_TIMEOUT*` is set, it
+limits the amount of time a client has to send the whole HTTP request.
+Otherwise, HAProxy will shut down the connection.
+
+Setting the environment variable allows the information to be captured as part
+of the router's deployment configuration and does not require manual
+modification of the template, whereas manually adding the HAProxy setting
+requires you to rebuild the router pod and maintain your router template file
+with the change.
```

Comment:
I don't understand the phrase "and maintain your router template file with the change".  Something seems to be missing.